code/modules/clothing/suits/jacket.dm

```diff
@@ -9,6 +9,14 @@
 	cold_protection = CHEST|GROIN|ARMS
 	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
 	species_exception = list(/datum/species/golem)
+	
+	/obj/item/clothing/suit/jacket
```

Comment:
Move all this to `modular_skyrat/modules/customization/modules/clothing/suits/jacket.dm`. The folder is there but you just need to make a new file. Ensure the new file is ticked once you've made it.